src/home/views.py

```
@@ -19,3 +19,33 @@
+def test_ai_function(user_id, posts_data):
+    # Funzione con diversi problemi per testare l'AI Review
+
+    from django.contrib.auth.models import User
+
+    # Problema 1: Query N+1 potential
+    user = User.objects.get(id=user_id)  # Dovrebbe essere get_object_or_404
+
+    # Problema 2: Lista invece di comprehension
+    result_posts = []
+    for post in posts_data:
+        if post["active"] == True:  # Dovrebbe essere 'is True'
```

Flake8 (7.2.0)

[error] comparison to True should be 'if cond is True:' or 'if cond:'

(E712)